### PR TITLE
Update Access Handle "Trying It Out" instructions

### PR DESCRIPTION
**Pull request by fivedots**
This PR updates the documentation to reflect the change in the feature flag name, and to point to our Origin Trial.

### CODE REVIEW

File: AccessHandle.md [hunk 1]
```diff
@@ -215,10 +215,18 @@ convenient, both from a specification and performance point of view.
 
 ## Trying It Out
 
-A prototype of access handles is available in Chrome Canary (from at least
-version `94.0.4597.0`). To enable it, launch Chrome with the
+A prototype of access handles is available in Chrome. If you're using version
+95 or higher, you can enable it by launching Chrome with the
+`--enable-blink-features=FileSystemAccessAccessHandle` flag or enabling
+"Experimental Web Platform features" in "chrome://flags". If you're using
+version 94, launch chrome with the
```
Comment (rstz) [inline AccessHandle.md:222]: Nit: Chrome

File: AccessHandle.md [hunk 2]
```diff
@@ -215,10 +215,18 @@ convenient, both from a specification and performance point of view.
 
 ## Trying It Out
 
-A prototype of access handles is available in Chrome Canary (from at least
-version `94.0.4597.0`). To enable it, launch Chrome with the
+A prototype of access handles is available in Chrome. If you're using version
```
Comment (rstz) [inline AccessHandle.md:218]: Maybe clarify that only sync access handles are available? Also, the term "sync access handles" never really appears in the remainder of the doc, it's just the name of the method.

**Review by rstz — APPROVED** [review]
LGTM with two nits.

File: AccessHandle.md (hunk 2 above)
Comment (fivedots) [reply AccessHandle.md:218]: Added a note and clarified what sync access handles are.

**Pull request merged**